### PR TITLE
 CORTX-30265 - paramiko upgrade to 2.10.4

### PR DESCRIPTION
**Pull request by swanand-gadre**
# Problem Statement
- Problem statement

Paramiko package 2.7.X is vulnerable.
It needs to be upgraded to version 2.10.4
This PR is raised to upgrade this version.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

This PR is raised to upgrade paramiko package from 2.7.X to 2.10.4

# Coding
   Checklist for Author
-  [X] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [X] New/Affected tests are executed on Latest Build
-  [X] Attach test execution logs
-  [X] Collection tested and no collection error introduced (`pytest --local True --collect-only`)
This PR is raised to upgrade paramiko package from 2.7.X to 2.10.4

# Review Checklist 
  Checklist for Author
- [X] JIRA number/GitHub Issue added to PR
- [X] PR is self reviewed
- [X] Jira and state/status is updated and JIRA is updated with PR link
- [X] Check if the description is clear and explained

All test details are available in a document attached to this ticket -> https://jts.seagate.com/browse/CORTX-30265

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide
Not applicable

### CODE REVIEW

**Comment by swanand-gadre** [comment]
**Please approve only.
Do not merge this PR.**

**Comment by swanand-gadre** [comment]
Requesting PR approval (only approval , **PLEASE DO NOT MERGE YET**) -

Test results are attached in the document and in the logs here -> https://jts.seagate.com/browse/CORTX-30265
Please review test results and logs (as there are some failures in the test logs)
but my understanding is that those failures are not related to paramiko upgrade.

However please review and let me know if any queries before approval.

**Review by seagate-sarang-sawant — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]

**Pull request merged**
